### ISSUE TITLE
Feature Request: Use side panel option

### ISSUE THREAD

**Issue by andrewrynhard** [jeremyramin/terminal-plus#323]
Hello and thank you for your work. I really enjoy this package. It would be nice if we had an option to have the terminal on the side of the Atom window. For people like me, who have a wider screen, it would free up some vertical space if needed.

**Comment by asteroidice** [comment]
Yeah an option to have the terminal as a tab would let you accomplish this and pretty much any other configuration. 👍

**Comment by asteroidice** [comment]
Ha would you look at that! #9

**Comment by andrewrynhard** [comment]
Not quite the same. Toggle full screen? Collapse? Close all? It would be nice to have the same feature set, just on the side.

**Comment by lukeplausin** [comment]
+1 please!
